src/testutils/createMediaPicture.ts

```diff
@@ -2,8 +2,11 @@ import faker from 'faker'
 import { CaaSApi_Media_Picture } from '../types'
 import { createDataEntry } from './createDataEntry'
 
-export const createMediaPicture = (id = faker.datatype.uuid()): CaaSApi_Media_Picture => {
-  const base = createDataEntry(id)
+export const createMediaPicture = (
+  id = faker.datatype.uuid(),
+  locale = faker.random.locale()
```

Comment:
I recommend not using random properties (obviously unless it is inherently random like a UUID) in tests. Random properties only make sense in the context of property based testing, where you run thousands of permutations in a unit test. Used as it is here it just makes everything more complicated:

- unless you are very careful with logging you won't know which locale you actually used when a test fails
- you don't know what is actually being tested and you don't actually _think_ about which locales it makes sense to test
- In scenarios where something works for a subset of locales but doesn't for others you could get 'lucky' and get something merged to master before it actually gets caught in a test